jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/PrivateMessageDto.java

```diff
@@ -36,13 +36,11 @@
 public class PrivateMessageDto {
 
 
-    @NotBlank
-    @Size(min = PrivateMessage.MIN_TITLE_LENGTH, max = PrivateMessage.MAX_TITLE_LENGTH, message = "{title.length}")
+    @BbCodeAwareSize(min = PrivateMessage.MIN_TITLE_LENGTH, max = PrivateMessage.MAX_TITLE_LENGTH)
```

Comment:
Why did you added `@BbCodeAwareSize` and `@BbCodeNesting` to title? Title does not support bb-codes